.github/workflows/maven.yml

```diff
@@ -22,12 +22,34 @@ jobs:
           java-version: 8.0.x
       - name: Verify Format and License
         run: mvn spotless:check
+  legacy-build:
```

Comment:
I'd really rather not maintain two different build targets, it's a recipe for drift and cut/paste errors.

Is there a way that we can combine them together with a conditional depending on the java version, that should be possible.

Reply:
sure they can be merged into one conditional target, i updated https://github.com/kubernetes-client/java/pull/2459 to show how it works. @joshlong by any chance can you rebase again onto my latest commit?